saleor/order/emails.py

```diff
@@ -88,7 +96,7 @@ def send_order_confirmation(order_pk, redirect_url, user_pk=None):
         order=email_data["context"]["order"],
         user=None,
         user_pk=user_pk,
-        email_type=events.OrderEventsEmails.ORDER,
+        email_type=events.OrderEventsEmails.ORDER_CONFIRMATION,
```

Comment:
This is change is probably correct but we need to confirm with @dominik-zeglen if this doesn't break the events UI in the dashboard. I think we use these type fields for rendering there but I'm not sure.

Reply:
If we used this `OrderEventsEmails.ORDER`, we will have to open a task for dashboard to change it to `ORDER_CONFIRMATION`.

Reply:
Well, we do :)

Reply:
Then please open a task to change it in the dashboard or if this is a simple replacement, we can fix it right away.

Reply:
I created a task.